index.js

```diff
@@ -74,6 +74,35 @@ const getCredentialsFromLocalConfig = function(serviceLabel, credentials) {
   return creds;
 }
 
+/**
+* Helper function used to add credentials bound to cloud functions using wsk service bind
+*
+* @param {Object} theParams - parameters sent to service
+* @param {string} service - name of service in bluemix used to retrieve credentials, used for IAM instances
+* @param {string} serviceAltName - alternate name of service used for cloud foundry instances
+* @return {Object} - returns parameters modified to include credentials from service bind
+*/
+const addCredentialsFromServiceBind = function(theParams, service, serviceAltName) {
```

Comment:
The method should be `get` not `add`

Reply:
switch to `params` rather than `theParams`.
`serviceName` rather than `service`